vis.py

```diff
@@ -121,7 +121,7 @@
 
 else:
     st.markdown(f'''
-    <a href="{link}">
+    <a href="#">
```

Comment:
hmm wygląda jakbyś wywalił linka do sersitive.eu a to miało przenosić na stronę główną SERSitive ;p 
a z tego co pamietam to # mówi o tym, żeby wrócić na górę obecnej strony czy tam refreshować?